Library/Homebrew/test/support/helper/spec/shared_context/integration_test.rb

```diff
@@ -78,7 +78,8 @@ def brew(*args)
       "HOMEBREW_BREW_FILE"        => HOMEBREW_PREFIX/"bin/brew",
       "HOMEBREW_INTEGRATION_TEST" => command_id_from_args(args),
       "HOMEBREW_TEST_TMPDIR"      => TEST_TMPDIR,
-      "HOMEBREW_DEVELOPER"        => ENV["HOMEBREW_DEVELOPER"],
+      "HOMEBREW_DEVELOPER"        => "1",
```

Comment:
Why is this change needed?

Reply:
Similar as above, they are needed to ensure sub brew process to pick up correct ruby.